toqito/channel_metrics/fidelity_of_separability.py

```diff
@@ -178,7 +179,7 @@ def fidelity_of_separability(
     problem.add_constraint((picos.I(dim_r) @ sym_choi) * choi * (picos.I(dim_r) @ sym_choi) == choi)
 
     # PPT condition on Choi state
-    sys = []
+    sys = [] # type: List[int]
```

Comment:
this is an inline type annotation as requested by mypy

`Need type annotation for "sys" (hint: "sys: list[<type>] = ...")  [var-annotated]`